doc/EnvironmentVariables.md

```diff
@@ -7,6 +7,7 @@
 | ConfigRefreshFrequencySecs           | Interval at which the EdgeAgent config is refreshed from upstream                                            | int32                                         | 3600                       |
 | ConfigSource                         | twin / local, specifies where the deployment config should be read from                                      | twin, local                                   | twin                       |
 | CoolOffTimeUnitInSeconds             | Time span to wait between restart attempts on a module                                                       | 0-300                                         | 10                         |
+| DisableDeviceAnalyticsTelemetry      | Whether to disable sending basic telemetry about the device to Microsoft.                                    | bool                                          | false                      |
```

Comment:
I searched the code, and I only see this variable defined and implemented in the `release/1.2` branch.

Reply:
Yes, there is a test failure in the corresponding PR for main that I am working on fixing. As soon as that is resolved, the new code will be available on main as well.

Reply:
@onalante-msft I assume that main and 1.2 now match, correct?